api/Element.json

```diff
@@ -6069,7 +6069,8 @@
           "support": {
             "chrome": [
               {
-                "version_added": "71"
+                "version_added": "71",
+                "notes": "The unprefixed fullscreen API shipped in Chromium 71, the <code>Promise</code>-enabled version of the vendor-prefixed API already shipped in Chromium 69. See <a href='https://crbug.com/383813#c57'>bug 383813</a>."
```

Comment:
The webkit information should be [done like this](https://github.com/mdn/browser-compat-data/blob/master/api/Element.json#L2091).

The information about the promise should probably be another item in the array. I've opened #4468, but will not hold up merging of this PR.

Reply:
Chrome Android and WebView Android need to match whatever you do here.

[Here's where you look up](https://cs.chromium.org/chromium/src/android_webview/tools/system_webview_shell/test/data/webexposed/not-webview-exposed.txt) what is and is not in WebView.